deployment/cdk/opensearch-service-migration/cdk.context.json

```diff
@@ -29,6 +29,16 @@
     "vpcId": "<VPC_ID>",
     "reindexFromSnapshotServiceEnabled": true,
     "reindexFromSnapshotMaxShardSizeGiB": 80,
-    "trafficReplayerServiceEnabled": true
+
+    "// settingsForCaptureAndReplay": "Enable the below services for live traffic capture and replay",
+    "trafficReplayerServiceEnabled": false,
+
+    "// help trafficReplayerExtraArgs": "Increase speedup factor in order replay requests at a faster rate to catchup",
```

Comment:
Although I'm not sure how to document this succinctly but it would nice to indicate that this speedup factor only holds true up to the limitations of one replayer.